frontend/src/scenes/events/EventsTable.tsx

```diff
@@ -347,20 +349,63 @@ export function EventsTable({
                             alignItems: 'start',
                         }}
                     >
-                        <div style={{ display: 'flex', gap: '0.75rem', flexDirection: 'column', flexGrow: 1 }}>
-                            <EventName
-                                value={eventFilter}
-                                onChange={(value: string) => {
-                                    setEventFilter(value || '')
+                        <div style={{ display: 'flex', gap: '0.75rem', flexDirection: 'column' }}>
+                            <div
+                                style={{
+                                    display: showFilters ? undefined : 'none',
+                                    maxWidth: 700,
+                                    padding: '1rem',
+                                    border: 'var(--border) solid 1px',
+                                    flex: 1,
+                                    borderRadius: 'var(--border)',
                                 }}
-                            />
-                            <PropertyFilters
-                                propertyFilters={properties}
-                                onChange={setProperties}
-                                pageKey={pageKey}
-                                style={{ marginBottom: 0 }}
-                                eventNames={eventFilter ? [eventFilter] : []}
-                            />
+                            >
+                                <div>
+                                    <Typography.Text strong style={{ marginBottom: '1rem' }}>
+                                        Filter by event
+                                        <Tooltip title="Show all occurrences of a specific event.">
+                                            <InfoCircleOutlined
+                                                className="info-icon ml-05"
+                                                style={{ color: 'var(--primary)' }}
```

Comment:
The info icons shouldn't have the primary color, as they aren't actionable (nor the primary part of this UI).